source/tutorials/multi-boot/dual-boot-win.rst

```diff
@@ -237,10 +247,135 @@ If you prefer not to use your BIOS to load the :guilabel:`Boot Menu` and select
 #. You should be presented with the :command:`systemd-boot` menu, as shown
    below.
 
+   .. rst-class:: dropshadow
+
    .. figure:: ../../_figures/multi-boot/dual-boot-win-04.png
       :scale: 85%
       :alt: systemd-boot menu
 
       Figure 5: systemd-boot menu
 
 
+Alternative: Install Windows 10 OS After |CL|
+*********************************************
+
+The following alternative guide shows how to install Windows 10 OS adjacent
+to an existing |CL| installation.
+
+Prerequisites
+*************
+
+* |CL| is already installed.  
+* There is unallocated disk space available. If the entire disk has been
+  allocated to |CL|, then shrink the root partition to make space for 
+  Windows 10 OS.
+
+Install Windows 10 OS
+=====================
+
+#. Start the Windows installer and follow the prompts.
+
+#. At the Type of installation screen, choose 
+   :guilabel:`Custom: Install Windows only (advanced)`. See Figure 6.
+
+   .. rst-class:: dropshadow
+
+   .. figure:: ../../_figures/multi-boot/dual-boot-win-06.png
+      :scale: 85%
+      :alt: Windows - Choose installation type
+
+      Figure 6: Windows - Choose installation type
+
+#. Select :guilabel:`Unallocated Space` and create a new partition of 
+   the desired size. For this example, we will use the entire unallocated 
+   space. See Figure 7.
+
+   .. rst-class:: dropshadow
+
+   .. figure:: ../../_figures/multi-boot/dual-boot-win-07.png
+      :scale: 85%
+      :alt: Windows - Create new partition
+
+      Figure 7: Windows - Create new partition
+
+  .. note::
```

Comment:
Minor stuff. The note here needs to be left-aligned exactly with the `.. figure::` that precedes it. 
That's whats causing the numbering to restart, and showing the grey areas on GH render.